agents/plugins/mk_postgres.py

```diff
@@ -362,13 +363,25 @@ def run_sql_as_db_user(self,
     def get_psql_and_bin_path(self):
         # type: () -> Tuple[str, str]
         """This method returns the system specific psql interface binary as callable string"""
-
+        drives = win32api.GetLogicalDriveStrings()
```

Comment:
Please replace this with
`ensure_str(subprocess_check_output(["wmic", "logicaldisk", "get", "deviceid"]))`
I don't know the exact output format, you will have to play around a bit. It would probably make sense to move this to a separate functions, something like

```
def _logical_drives(self):
    return self._parse_wmic_logicaldisk(ensure_str(subprocess_check_output(["wmic", "logicaldisk", "get", "deviceid"])))

def _parse_wmic_logicaldisk(self, wmic_output):
    ...
```

Reply:
Thank you for your help in this matter. Turns out the format of the wmic output is slightly weird. It is actually starts with DeviceID and is followed by the drive names separated by two carriage returns and a line feed, additionally each line is padded to reach 10 characters in length.